slapp/data_selection/scripts/suite2p_array_job_example.pbs

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
Hmm... I feel like this should be in ophys_segmentation repo instead.

Reply:
moved to ``ophys_segmentation``.